src/time.cr

```diff
@@ -140,7 +140,15 @@ struct Time
     new(seconds: seconds + offset, nanoseconds: nanoseconds, kind: Kind::Local)
   end
 
-  def self.new(year, month, day, hour = 0, minute = 0, second = 0, nanosecond = 0, kind = Kind::Unspecified)
+  def self.new(year, month, day, hour = 0, minute = 0, second = 0, *, millisecond = 0, kind = Kind::Unspecified)
```

Comment:
This constructor should never be called if `millisecond` isn't specified (no conversion required) so simply remove the default value and specs should pass.